### PR TITLE
Update MLOps Python article with additional info from microsoft/MLOpsPython

### PR DESCRIPTION
**Pull request by tcare**
We have evolved the microsoft/MLOpsPython README.md with additional information on the architecture of MLOpsPython. We want to remove the majority of the architecture text from there and link to this article instead. See https://github.com/microsoft/MLOpsPython/pull/220 for more details.

### CODE REVIEW

**Comment by tcare** [comment]
@dtzar

**Comment by PRMerger13** [comment]
@tcare : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/reference-architectures/ai/mlops-python.md [hunk 1]
```diff
@@ -57,9 +57,13 @@ The next sections describe each of these pipelines.
...
+- **Code quality.** These tests ensure that the code conforms to the standards of the team.
 
-- **Data test.** These tests verify that the data samples conform to the expected schema and distribution. Customize this test for other use cases and run it as a separate data sanity pipeline that gets triggered as new data arrives. For example, move the data test task to a data ingestion pipeline so you can test it earlier.
+- **Unit test.** These tests make sure the code works, has adequate code coverage, and is stable.
+
+- **Data test.** These tests verify that the data samples conform to the expected schema and distribution. Customize this test for other use cases and run it as a separate data sanity pipeline that gets triggered as new data arrives. For example, move the data test task to a *data ingestion pipeline* so you can test it earlier.
+
+**Note:** CI/CD concepts for data ingestion pipelines are not covered in this article. For more information on the architecture and best practices for CI/CD of a data ingestion pipeline, see [DevOps for a data ingestion pipeline](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-cicd-data-ingestion).
```
Comment (dtzar) [inline docs/reference-architectures/ai/mlops-python.md]: Consider changing the first sentence to: "You should consider enabling DevOps practices for the data used to train the machine learning models, but this is not covered in this article."

**Comment by PRMerger18** [comment]
@tcare : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by dtzar** [comment]
@praneet22 Thanks for the approval.  I think you just need to post a comment **#sign-off** so it will merge.

**Comment by tcare** [comment]
Actually I might have to post it to the architecture-center-pr repo instead for automerge to enable.. looking at that now

**Comment by PRMerger20** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
Hi @tcare - The Aquent PR review team merges public repo PRs after they are signed off by the listed author. Twice a day, weekdays, we look for PRs that are ready to merge. Merging now...

**Comment by tcare** [comment]
Thanks Kristine :)

**Comment by ktoliver** [comment]
Sure thing--it's described more succinctly here: https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-manage-act-guidelines-public-pr?branch=master#workflow-overview

**Pull request merged**